api/models/Team.ts

```diff
@@ -42,4 +48,14 @@ export class Team extends BaseEntity {
     @OneToMany(() => Submission, (submissions) => submissions.team)
     submissions!: Submission[];
 
+    async getElimination (round: Round): Promise<EliminationDetails | undefined> {
```

Comment:
shouldnt this return `Promise<EliminationDetails>`? it wont ever be undefined i believe?